tst/CTA.Rules.Test/Actions/ClassActionsTests.cs

```diff
@@ -175,6 +175,148 @@ public void RemoveConstructorBaseClass()
+
+
+            StringAssert.Contains("Task<string>", nodeWithExpression.ToFullString());
+        }
+
+        [Test]
+        public void CommentMethod()
+        {
+            string methodName = "Invoke";
+            var methodNode = SyntaxFactory.MethodDeclaration(SyntaxFactory.ParseTypeName("void"), methodName);
+            var nodeWithMethod = _node.AddMembers(methodNode);
+
+            var CommentMethodeFunc = _classActions.GetCommentMethodAction(methodName);
+            var nodeWithExpression = CommentMethodeFunc(_syntaxGenerator, nodeWithMethod);
+
+            StringAssert.Contains("/*voidInvoke()*/", nodeWithExpression.ToFullString());
+        }
+
+        [Test]
+        public void AddCommentsToMethod()
+        {
+            string methodName = "Invoke";
+            string comment = "This method is deprecated";
+            var methodNode = SyntaxFactory.MethodDeclaration(SyntaxFactory.ParseTypeName("void"), methodName);
+            var nodeWithMethod = _node.AddMembers(methodNode);
+
+            var AddCommentsToMethodFunc = _classActions.GetAddCommentsToMethodAction(methodName, comment);
+            var nodeWithExpression = AddCommentsToMethodFunc(_syntaxGenerator, nodeWithMethod);
+
+            StringAssert.Contains("/* Added by CTA: This method is deprecated */", nodeWithExpression.ToFullString());
+        }
+
+        [Test]
+        public void AddExpressionToMethod()
+        {
+            string methodName = "Invoke";
+            string expression = "await _next.Invoke(context);";
+            var methodNode = SyntaxFactory.MethodDeclaration(SyntaxFactory.ParseTypeName("void"), methodName).AddBodyStatements();
+            var nodeWithMethod = _node.AddMembers(methodNode);
+
+            var AddExpressionToMethodFunc = _classActions.GetAddExpressionToMethodAction(methodName, expression);
+            var nodeWithExpression = AddExpressionToMethodFunc(_syntaxGenerator, nodeWithMethod);
+
+            StringAssert.Contains("await _next.Invoke(context);", nodeWithExpression.ToFullString());
+        }
+
+        [Test]
+        public void AddParametersToMethod()
+        {
+            string methodName = "Invoke";
+            string types = "HttpContext, string";
+            string identifiers = " context, value";
+
+            var methodNode = SyntaxFactory.MethodDeclaration(SyntaxFactory.ParseTypeName("void"), methodName).AddBodyStatements();
+            var nodeWithMethod = _node.AddMembers(methodNode);
+
+            var AddParametersToMethodFunc = _classActions.GetAddParametersToMethodAction(methodName, types, identifiers);
+            var nodeWithExpression = AddParametersToMethodFunc(_syntaxGenerator, nodeWithMethod);
+
+            StringAssert.Contains("HttpContext context", nodeWithExpression.ToFullString());
+            StringAssert.Contains("string value", nodeWithExpression.ToFullString());
+        }
```

Comment:
Something that might help here (and with other tests) is parsing a string into a method declaration node, e.g.:
```csharp
var methodString =  
@"public string MyStringMethod(string s)
{
    return string.empty;
}");
var methodDeclaration = SyntaxFactory.ParseMemberDeclaration(methodString);

//
// Modify the method here...
//

var expectedString =  
@"public string MyStringMethod(string s, string theNewParameter)
{
    return string.empty;
}");
Assert.AreEqual(expectedString, methodDeclaration.ToFullString());
```

Will make the test easier to write and understand.